### PR TITLE
[Upgrade] Go-Ethereum release v1.9.21

### PR DESCRIPTION
**Pull request by quorumbot**
### Plan & Analyse

- [x] Review the Release Notes
- [x] Review PRs in the section below

As you review, list extra changes and/or tests to be implemented to ensure compatibility with GoQuorum specific features.

### Build & Test

- [x] Pull and checkout PR branch locally, then merge GoQuorum `master` into this branch
- [x] Resolve conflicts, taking into account the prior analysis
- [x] Implement required changes until lint passes
- [x] Implement required changes until all unit tests pass
- [x] Implement required changes until acceptance tests pass
- [x] Implement extra changes and/or tests
- [x] Verify any left TODOs in the code

### Extra Changes & Tests

- [x] debug_traceCall : it's a new JSON-RPC API method that allows to call a contract with different arguments than debug_traceTransaction. This has been adapted so that it is compatible with private contract. [Link to the source code](https://github.com/ConsenSys/quorum/pull/1211/files#diff-fa406a35f903f7a2826290aba52257315413fda284257e152f9a9b0fcecef91aR833)
- [x] did not upgraded leveldb and protobuf because of our version of etcd. These upgrades cause some acceptance tests failures. [Link to the commit](https://github.com/ConsenSys/quorum/pull/1211/commits/3f23db905de2233754740080294a98a59513f45b)

### Left TODOs

- [x] We should upgrade the [etcd we use](github.com/Consensys/etcd) v3.3.13-quorum197+incompatible so that it is compatible with protobuf 1.4.2. It will unblock the usage of a newer leveldb version. We could [align the versions](https://github.com/ConsenSys/quorum/pull/1211/commits/3f23db905de2233754740080294a98a59513f45b) with original go-ethereum
  - Newer version of leveldb will fix a problem with released snapshots, have better performance, and a new configuration set. [See PR](https://github.com/ethereum/go-ethereum/pull/21448)

## Go-Ethereum Release: Tavum (v1.9.21)

* Version: v1.9.21
* Published: 2020-09-09T08:25:11Z

### Release notes 

Geth v1.9.21 is a regular maintenance release, the highlights being the removal of whisper, better call tracing and multiple memory stability fixes during fast sync to both stabilize usage as well as to fix a memory leak that lead to crashers before.

- Remove Whisper as promised a couple months ago ([#21487](https://github.com/ethereum/go-ethereum/pull/21487), [#21526](https://github.com/ethereum/go-ethereum/pull/21526), [#21527](https://github.com/ethereum/go-ethereum/pull/21527))!
- Minor user experience polishes around legacy Ledger derivation paths ([#21517](https://github.com/ethereum/go-ethereum/pull/21517)).
- Implement arbitrary call tracing via `debug_traceCall` on top of arbitrary blocks ([#21338](https://github.com/ethereum/go-ethereum/pull/21338)).
- Expose internal transaction revertals and revert reason in the `call_tracer` tracer ([#21387](https://github.com/ethereum/go-ethereum/pull/21387)). 
- Cap the number of in-memory trie nodes during fast sync, fix crasher memory leak ([#21491](https://github.com/ethereum/go-ethereum/pull/21491)).
- Prepare the trie syncer for path-based operation to support the upcoming `snap` sync ([#21504](https://github.com/ethereum/go-ethereum/pull/21504)).
- Limit the cached data in the downloader more aggressively to avoid memory fluctuations ([#21366](https://github.com/ethereum/go-ethereum/pull/21366)).
- Fix the simulated chain to not allow changing block timestamps if transactions were included ([#21334](https://github.com/ethereum/go-ethereum/pull/21334)).
- Fix an ABI parser issue around tuples ([#21501](https://github.com/ethereum/go-ethereum/pull/21501)).

For a full rundown of the changes please consult the Geth 1.9.21 [release milestone](https://github.com/ethereum/go-ethereum/milestone/107?closed=1)

---

As with all our previous releases, you can find the:

- Pre-built binaries for all platforms on our [downloads page](https://geth.ethereum.org/downloads/).
- Docker images published under [`ethereum/client-go`](https://cloud.docker.com/u/ethereum/repository/docker/ethereum/client-go).
- Ubuntu packages in our [Launchpad PPA repository](https://launchpad.net/~ethereum/+archive/ubuntu/ethereum).
- OSX packages in our [Homebrew Tap repository](https://github.com/ethereum/homebrew-ethereum).

### Legend

File Stats: (A) Added, (M) Modified and (R) Removed

Line Stats: (A) Added and (R) Removed

Assessment:

* ✅ No conflict expected
* ⚠ Review required to assess changes
* ‼️ Conflicts expected and review required

### 19 Pull Requests

| 🔍 | Link | Title | File Stats<br>M/A/R | Packages changed<br>(files changed) | Line Stats<br>A/R | Top 5 Changed Files<br>(lines changed) |
| :--- | :--- | :--- | :--- | :--- | :--- | :--- |
| ‼️ | [#21334](https://github.com/ethereum/go-ethereum/pull/21334) | ``accounts/abi/bind/backends: Disallow AdjustTime for non-empty blocks`` | 2/0/0<br> | ``accounts/abi/bind/backends`` (2)<br> | <span class="text-green">44</span>/<span class="text-red">5</span><br> | ``accounts/abi/bind/backends/simulated_test.go`` (82)<br>``accounts/abi/bind/backends/simulated.go`` (16)<br> |
| ⚠️ | [#21490](https://github.com/ethereum/go-ethereum/pull/21490) | ``eth: updated comments`` | 3/0/0<br> | ``eth`` (3)<br> | <span class="text-green">3</span>/<span class="text-red">3</span><br> | ``eth/api.go`` (4)<br>``eth/api_tracer.go`` (4)<br>``eth/bloombits.go`` (4)<br> |
| ‼️ | [#21448](https://github.com/ethereum/go-ethereum/pull/21448) | ``go.mod | goleveldb latest update`` | 2/0/0<br> | ``go.mod`` (1)<br>``go.sum`` (1)<br> | <span class="text-green">54</span>/<span class="text-red">17</span><br> | ``go.sum`` (120)<br>``go.mod`` (22)<br> |
| ✅ | [#21491](https://github.com/ethereum/go-ethereum/pull/21491) | ``core/state, eth, trie: stabilize memory use, fix memory leak`` | 7/0/0<br> | ``trie`` (4)<br>``core/state`` (2)<br>``eth/downloader`` (1)<br> | <span class="text-green">58</span>/<span class="text-red">29</span><br> | ``trie/sync.go`` (120)<br>``eth/downloader/downloader.go`` (22)<br>``core/state/sync.go`` (12)<br>``trie/committer.go`` (8)<br>``core/state/statedb.go`` (4)<br> |
| ⚠️ | [#21387](https://github.com/ethereum/go-ethereum/pull/21387) | ``eth/tracers: revert reason in call_tracer + error for failed internal calls`` | 3/2/0<br> | ``eth/tracers/internal/tracers`` (2)<br>``eth/tracers/testdata`` (2)<br>``eth/tracers`` (1)<br> | <span class="text-green">172</span>/<span class="text-red">15</span><br> | ``eth/tracers/testdata/call_tracer_inner_instafail.json`` (144)<br>``eth/tracers/testdata/call_tracer_revert_reason.json`` (128)<br>``eth/tracers/tracers_test.go`` (48)<br>``eth/tracers/internal/tracers/call_tracer.js`` (30)<br>``eth/tracers/internal/tracers/assets.go`` (24)<br> |
| ✅ | [#21497](https://github.com/ethereum/go-ethereum/pull/21497) | ``rpc: fix issue with null JSON-RPC messages`` | 3/0/0<br> | ``rpc/testdata`` (2)<br>``rpc`` (1)<br> | <span class="text-green">16</span>/<span class="text-red">3</span><br> | ``rpc/json.go`` (26)<br>``rpc/testdata/invalid-batch.js`` (6)<br>``rpc/testdata/invalid-nonobj.js`` (6)<br> |
| ✅ | [#21501](https://github.com/ethereum/go-ethereum/pull/21501) | ``accounts/abi: fix a bug in getTypeSize method`` | 2/0/0<br> | ``accounts/abi`` (2)<br> | <span class="text-green">37</span>/<span class="text-red">1</span><br> | ``accounts/abi/type_test.go`` (72)<br>``accounts/abi/type.go`` (4)<br> |
| ✅ | [#21503](https://github.com/ethereum/go-ethereum/pull/21503) | ``internal: fix personal.sign()`` | 1/0/0<br> | ``console`` (1)<br> | <span class="text-green">2</span>/<span class="text-red">2</span><br> | ``console/bridge.go`` (8)<br> |
| ✅ | [#21455](https://github.com/ethereum/go-ethereum/pull/21455) | ``"Downloader queue stats" is now provided once per minute`` | 1/0/0<br> | ``eth/downloader`` (1)<br> | <span class="text-green">1</span>/<span class="text-red">1</span><br> | ``eth/downloader/queue.go`` (4)<br> |
| ‼️ | [#21432](https://github.com/ethereum/go-ethereum/pull/21432) | ``go.mod: goja update`` | 2/0/0<br> | ``go.mod`` (1)<br>``go.sum`` (1)<br> | <span class="text-green">3</span>/<span class="text-red">3</span><br> | ``go.sum`` (8)<br>``go.mod`` (4)<br> |
| ✅ | [#21366](https://github.com/ethereum/go-ethereum/pull/21366) | ``eth/downloader: change intial download size`` | 5/0/0<br> | ``eth/downloader`` (5)<br> | <span class="text-green">26</span>/<span class="text-red">24</span><br> | ``eth/downloader/downloader_test.go`` (48)<br>``eth/downloader/queue.go`` (28)<br>``eth/downloader/queue_test.go`` (12)<br>``eth/downloader/downloader.go`` (8)<br>``eth/downloader/testchain_test.go`` (4)<br> |
| ✅ | [#21504](https://github.com/ethereum/go-ethereum/pull/21504) | ``core, eth, trie: prepare trie sync for path based operation`` | 6/0/0<br> | ``trie`` (4)<br>``core/state`` (1)<br>``eth/downloader`` (1)<br> | <span class="text-green">480</span>/<span class="text-red">105</span><br> | ``eth/downloader/statesync.go`` (306)<br>``trie/sync_test.go`` (298)<br>``core/state/sync_test.go`` (254)<br>``trie/trie.go`` (168)<br>``trie/sync.go`` (132)<br> |
| ⚠️ | [#21338](https://github.com/ethereum/go-ethereum/pull/21338) | ``eth: added trace_call to trace on top of arbitrary blocks`` | 3/0/0<br> | ``eth`` (2)<br>``internal/web3ext`` (1)<br> | <span class="text-green">54</span>/<span class="text-red">8</span><br> | ``eth/api_tracer.go`` (98)<br>``eth/api.go`` (14)<br>``internal/web3ext/web3ext.go`` (12)<br> |
| ‼️ | [#21514](https://github.com/ethereum/go-ethereum/pull/21514) | ``accounts/abi/bid/backends:`` | 2/0/0<br> | ``accounts/abi/bind/backends`` (2)<br> | <span class="text-green">76</span>/<span class="text-red">76</span><br> | ``accounts/abi/bind/backends/simulated.go`` (264)<br>``accounts/abi/bind/backends/simulated_test.go`` (40)<br> |
| ✅ | [#21487](https://github.com/ethereum/go-ethereum/pull/21487) | ``whisper: remove whisper`` | 6/0/24<br> | ``whisper/whisperv6`` (17)<br>``mobile`` (3)<br>``cmd/geth`` (3)<br>``whisper/mailserver`` (2)<br>``cmd/wnode`` (1)<br>``tests/fuzzers/whisperv6`` (1)<br>``whisper/shhclient`` (1)<br>``cmd/utils`` (1)<br>``tests/fuzzers/whisperv6/corpus`` (1)<br> | <span class="text-green">17</span>/<span class="text-red">5826</span><br> | ``whisper/whisperv6/filter_test.go`` (1672)<br>``cmd/wnode/main.go`` (1546)<br>``whisper/whisperv6/api.go`` (1186)<br>``whisper/whisperv6/message_test.go`` (942)<br>``whisper/whisperv6/message.go`` (710)<br> |
| ⚠️ | [#21517](https://github.com/ethereum/go-ethereum/pull/21517) | ``accounts/usbwallet, signer/core: show accounts from ledger legacy derivation paths`` | 2/0/0<br> | ``accounts/usbwallet`` (1)<br>``signer/core`` (1)<br> | <span class="text-green">30</span>/<span class="text-red">17</span><br> | ``signer/core/api.go`` (66)<br>``accounts/usbwallet/wallet.go`` (28)<br> |
| ⚠️ | [#21526](https://github.com/ethereum/go-ethereum/pull/21526) | ``build: remove wnode from the list of packages binaries`` | 1/0/0<br> | ``build`` (1)<br> | <span class="text-green">0</span>/<span class="text-red">5</span><br> | ``build/ci.go`` (10)<br> |
| ✅ | [#21527](https://github.com/ethereum/go-ethereum/pull/21527) | ``.github: remove whisper from CODEOWNERS`` | 1/0/0<br> | ``.github`` (1)<br> | <span class="text-green">0</span>/<span class="text-red">1</span><br> | ``.github/CODEOWNERS`` (2)<br> |
| ⚠️ | [#21534](https://github.com/ethereum/go-ethereum/pull/21534) | ``params: update CHTs for v1.9.21 release`` | 1/0/0<br> | ``params`` (1)<br> | <span class="text-green">16</span>/<span class="text-red">16</span><br> | ``params/config.go`` (64)<br> |

### 72 Changed files

| 🔍 | File | Lines Changed | Linked PR |
| :--- | :--- | :--- | :--- |
| ✅ | ``whisper/whisperv6/whisper.go`` | 2280 |  |
| ✅ | ``whisper/whisperv6/whisper_test.go`` | 1856 |  |
| ✅ | ``whisper/whisperv6/filter_test.go`` | 1672 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``cmd/wnode/main.go`` | 1546 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/whisperv6/api.go`` | 1186 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/whisperv6/message_test.go`` | 942 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/whisperv6/message.go`` | 710 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/whisperv6/envelope.go`` | 560 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/whisperv6/peer.go`` | 536 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/whisperv6/filter.go`` | 524 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/mailserver/server_test.go`` | 470 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/mailserver/mailserver.go`` | 418 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/whisperv6/benchmarks_test.go`` | 416 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``mobile/shhclient.go`` | 390 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/shhclient/client.go`` | 386 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``eth/downloader/statesync.go`` | 306 | [#21504](https://github.com/ethereum/go-ethereum/pull/21504)<br> |
| ✅ | ``trie/sync_test.go`` | 298 | [#21504](https://github.com/ethereum/go-ethereum/pull/21504)<br> |
| ‼️ | ``accounts/abi/bind/backends/simulated.go`` | 280 | [#21514](https://github.com/ethereum/go-ethereum/pull/21514)<br>[#21334](https://github.com/ethereum/go-ethereum/pull/21334)<br> |
| ✅ | ``whisper/whisperv6/topic_test.go`` | 268 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``core/state/sync_test.go`` | 254 | [#21504](https://github.com/ethereum/go-ethereum/pull/21504)<br> |
| ✅ | ``trie/sync.go`` | 240 | [#21491](https://github.com/ethereum/go-ethereum/pull/21491)<br>[#21504](https://github.com/ethereum/go-ethereum/pull/21504)<br> |
| ✅ | ``mobile/types.go`` | 186 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/whisperv6/doc.go`` | 184 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/whisperv6/envelope_test.go`` | 182 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``tests/fuzzers/whisperv6/whisper-fuzzer.go`` | 180 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/whisperv6/gen_newmessage_json.go`` | 180 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``trie/trie.go`` | 172 | [#21491](https://github.com/ethereum/go-ethereum/pull/21491)<br>[#21504](https://github.com/ethereum/go-ethereum/pull/21504)<br> |
| ✅ | ``whisper/whisperv6/gen_message_json.go`` | 168 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``eth/tracers/testdata/call_tracer_inner_instafail.json`` | 144 | [#21387](https://github.com/ethereum/go-ethereum/pull/21387)<br> |
| ✅ | ``whisper/whisperv6/gen_criteria_json.go`` | 132 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``whisper/whisperv6/api_test.go`` | 128 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ‼️ | ``go.sum`` | 128 | [#21432](https://github.com/ethereum/go-ethereum/pull/21432)<br>[#21448](https://github.com/ethereum/go-ethereum/pull/21448)<br> |
| ✅ | ``eth/tracers/testdata/call_tracer_revert_reason.json`` | 128 | [#21387](https://github.com/ethereum/go-ethereum/pull/21387)<br> |
| ‼️ | ``accounts/abi/bind/backends/simulated_test.go`` | 122 | [#21514](https://github.com/ethereum/go-ethereum/pull/21514)<br>[#21334](https://github.com/ethereum/go-ethereum/pull/21334)<br> |
| ✅ | ``whisper/whisperv6/topic.go`` | 112 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ‼️ | ``eth/api_tracer.go`` | 102 | [#21490](https://github.com/ethereum/go-ethereum/pull/21490)<br>[#21338](https://github.com/ethereum/go-ethereum/pull/21338)<br> |
| ✅ | ``accounts/abi/type_test.go`` | 72 | [#21501](https://github.com/ethereum/go-ethereum/pull/21501)<br> |
| ⚠️ | ``signer/core/api.go`` | 66 | [#21517](https://github.com/ethereum/go-ethereum/pull/21517)<br> |
| ⚠️ | ``params/config.go`` | 64 | [#21534](https://github.com/ethereum/go-ethereum/pull/21534)<br> |
| ✅ | ``whisper/whisperv6/config.go`` | 62 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ‼️ | ``cmd/utils/flags.go`` | 54 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ‼️ | ``cmd/geth/config.go`` | 54 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ⚠️ | ``eth/downloader/downloader_test.go`` | 48 | [#21366](https://github.com/ethereum/go-ethereum/pull/21366)<br> |
| ⚠️ | ``eth/tracers/tracers_test.go`` | 48 | [#21387](https://github.com/ethereum/go-ethereum/pull/21387)<br> |
| ✅ | ``eth/downloader/queue.go`` | 32 | [#21455](https://github.com/ethereum/go-ethereum/pull/21455)<br>[#21366](https://github.com/ethereum/go-ethereum/pull/21366)<br> |
| ⚠️ | ``eth/downloader/downloader.go`` | 30 | [#21491](https://github.com/ethereum/go-ethereum/pull/21491)<br>[#21366](https://github.com/ethereum/go-ethereum/pull/21366)<br> |
| ✅ | ``eth/tracers/internal/tracers/call_tracer.js`` | 30 | [#21387](https://github.com/ethereum/go-ethereum/pull/21387)<br> |
| ⚠️ | ``accounts/usbwallet/wallet.go`` | 28 | [#21517](https://github.com/ethereum/go-ethereum/pull/21517)<br> |
| ‼️ | ``go.mod`` | 26 | [#21432](https://github.com/ethereum/go-ethereum/pull/21432)<br>[#21448](https://github.com/ethereum/go-ethereum/pull/21448)<br> |
| ⚠️ | ``rpc/json.go`` | 26 | [#21497](https://github.com/ethereum/go-ethereum/pull/21497)<br> |
| ✅ | ``eth/tracers/internal/tracers/assets.go`` | 24 | [#21387](https://github.com/ethereum/go-ethereum/pull/21387)<br> |
| ✅ | ``mobile/geth.go`` | 20 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ‼️ | ``eth/api.go`` | 18 | [#21490](https://github.com/ethereum/go-ethereum/pull/21490)<br>[#21338](https://github.com/ethereum/go-ethereum/pull/21338)<br> |
| ‼️ | ``cmd/geth/consolecmd_test.go`` | 16 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ✅ | ``core/state/sync.go`` | 12 | [#21491](https://github.com/ethereum/go-ethereum/pull/21491)<br> |
| ✅ | ``eth/downloader/queue_test.go`` | 12 | [#21366](https://github.com/ethereum/go-ethereum/pull/21366)<br> |
| ✅ | ``trie/secure_trie.go`` | 12 | [#21504](https://github.com/ethereum/go-ethereum/pull/21504)<br> |
| ⚠️ | ``internal/web3ext/web3ext.go`` | 12 | [#21338](https://github.com/ethereum/go-ethereum/pull/21338)<br> |
| ⚠️ | ``build/ci.go`` | 10 | [#21526](https://github.com/ethereum/go-ethereum/pull/21526)<br> |
| ✅ | ``trie/committer.go`` | 8 | [#21491](https://github.com/ethereum/go-ethereum/pull/21491)<br> |
| ✅ | ``console/bridge.go`` | 8 | [#21503](https://github.com/ethereum/go-ethereum/pull/21503)<br> |
| ✅ | ``rpc/testdata/invalid-nonobj.js`` | 6 | [#21497](https://github.com/ethereum/go-ethereum/pull/21497)<br> |
| ✅ | ``rpc/testdata/invalid-batch.js`` | 6 | [#21497](https://github.com/ethereum/go-ethereum/pull/21497)<br> |
| ⚠️ | ``eth/bloombits.go`` | 4 | [#21490](https://github.com/ethereum/go-ethereum/pull/21490)<br> |
| ✅ | ``trie/trie_test.go`` | 4 | [#21491](https://github.com/ethereum/go-ethereum/pull/21491)<br> |
| ⚠️ | ``cmd/geth/usage.go`` | 4 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |
| ⚠️ | ``core/state/statedb.go`` | 4 | [#21491](https://github.com/ethereum/go-ethereum/pull/21491)<br> |
| ✅ | ``accounts/abi/type.go`` | 4 | [#21501](https://github.com/ethereum/go-ethereum/pull/21501)<br> |
| ✅ | ``eth/downloader/testchain_test.go`` | 4 | [#21366](https://github.com/ethereum/go-ethereum/pull/21366)<br> |
| ⚠️ | ``params/version.go`` | 4 |  |
| ✅ | ``.github/CODEOWNERS`` | 2 | [#21527](https://github.com/ethereum/go-ethereum/pull/21527)<br> |
| ✅ | ``tests/fuzzers/whisperv6/corpus/009c5adfa4fd685caef58e1ce932fa7fb209730a`` | 0 | [#21487](https://github.com/ethereum/go-ethereum/pull/21487)<br> |

### CODE REVIEW

**Review by nmvalera — APPROVED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -44,7 +44,6 @@ require (
 	github.com/golang/mock v1.4.3
 	github.com/golang/protobuf v1.3.4
```
Comment (ricardolyn) [inline go.mod:45]: should we add a comment here to maintain the old version?

**Review by ricardolyn — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (baptiste-b-pegasys) [reply go.mod:45]: do you mean add comment in order to keep the geth's version?
I prefer to prepare another PR to solve the remaining TODOs, which is a revert of the last commit.

Comment (ricardolyn) [reply go.mod:45]: @baptiste-b-pegasys can you create an issue to track this, so we are aware we need to put back the original version from geth?

Comment (ricardolyn) [reply go.mod:45]: in the meantime, I can merge this.

**Pull request merged**